app/views/planning_applications/_decision_notice.html.erb

```diff
@@ -20,88 +20,90 @@
-            <th scope="col" class="govuk-table__header app-custom-class">Date received</th>
-          </tr>
-        </thead>
-        <tbody class="govuk-table__body">
-          <% @planning_application.documents.for_display.each do |document| %>
-            <% document.numbers.split(",").each do |number| %>
-                <tr class="govuk-table__row">
-                  <td class="govuk-table__cell"><%= number %></td>
-                  <td class="govuk-table__cell">
-                    <% document.tags.each do |tag| %>
-                      <%= tag.upcase %>
-                    <% end %>
-                  </td>
-                  <td class="govuk-table__cell"><%= document.created_at.to_formatted_s(:day_month_year) %></td>
-                </tr>
-           <% end %>
-         <% end %>
-        </tbody>
-      </table>
-    <% else %>
-      <p class="govuk-body">
-        No plans specified
-      </p>
-    <% end %>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        <%= key %>
+      </dt>
+      <dd class="govuk-summary-list__value">
+        <%= value %>
+      </dd>
+    </div>
   <% end %>
-
-  <h3 class="govuk-heading-s">
-    Site location
+</dl>
+<p class="govuk-body">
+  <strong>
+    Use/development
+  </strong>
+</p>
+<p class="govuk-body">
+  We certify that on the date of the application, the <%= @planning_application.work_status %> use or operations described in the application and supporting plans were <%= @planning_application.granted? ? "lawful" : "not lawful" %> for the purposes of <%= @planning_application.work_status == "proposed" ? "S.192" : "S.191" %> of the Town and Country Planning Act 1990.
+</p>
+<p class="govuk-body govuk-!-margin-bottom-1">
+  <strong>
+    The proposal <%= @planning_application.granted? ? "complies" : "does not comply" %> due to the following reason(s):
+  </strong>
+</p>
+<p class="govuk-body">
+  <%= planning_application.public_comment %>
+</p>
+<% if @planning_application.granted? %>
+  <h3 class="govuk-heading-s govuk-!-margin-bottom-1">
+    This decision is based on the following approved plans:
   </h3>
-  <% if @planning_application.boundary_geojson.present? %>
-    <%= render "shared/location_map", locals: { geojson: @planning_application.boundary_geojson } %>
+  <% if planning_application.documents.for_display.present? %>
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header app-custom-class">Document reference</th>
+          <th scope="col" class="govuk-table__header app-custom-class">Description</th>
+          <th scope="col" class="govuk-table__header app-custom-class">Date received</th>
+        </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        <% @planning_application.documents.for_display.each do |document| %>
```

Comment:
can you make this an instance variable?

Reply:
have popped it into the planning application model instead. If this isn't good I feel we should just go back to what it was instead of changing 3 different controller methods and a setter just to save 2 words..